test/apiv2/10-images.at

```diff
@@ -343,4 +343,13 @@ t GET libpod/images/no-alias-for-sure/resolve 200 \
 t GET libpod/images/noCAPITALcharAllowed/resolve 400 \
   .cause="repository name must be lowercase"
 
+
+START=$(date +%s.%N)
+# test pull-error API response
+podman pull --retry 0 localhost:5000/idonotexist || true
+t GET "libpod/events?stream=false&since=$START"  200  \
+  'select(.status | contains("pull-error")).Action=pull-error' \
+  'select(.status | contains("pull-error")).Actor.Attributes.name=localhost:5000/idonotexist' \
+  'select(.status | contains("pull-error")).Actor.Attributes.error~.*connection refused'
```

Comment:
A more idiomatic way to write this would be
```
t GET "libpod/events?stream=false&since=$START"  200  \
  .status=pull-error \
  .Action=pull-error \
  .Actor.Attributes.name=localhost:5000/idonotexist \
  .Actor.Attributes.error~".*connection refused"
```

Reply:
I just copy pasted from another location, I will change it but maybe there was a reason for it.

Reply:
That dates to #15487 which I even reviewed and approved. I must've assumed there was a good reason for the `select`, and not had time to actually review in depth. My assumption is, greater familiarity with `jq` than with the apiv2 test conventions.